### PR TITLE
Lesson 2 - Assignment 1

### PR DESCRIPTION
**Pull request by paulfleurdelys**
Submitting calculator.js file.

### CODE REVIEW

**Comment by paulfleurdelys** [comment]
Submitting countdown.js file.

File: Lesson02/calculator.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+let x = 2;
+let y = 4;
+let z = 10;
+let a = -5;
+
+console.log("Addition, " + x + " plus " + y + " is equal to " + (x + y));
+console.log("Subtraction, " + y + " minus " + z + " is equal to " + (y - z));
+console.log("Multiplication, " + z + " times " + a + " is equal to " + (z * a));
+console.log("Division, " + x + " divided by " + y + " is equal to " + (x / y));
+console.log("Mod, " + z + " remainder " + y + " is equal to " + (z % y));
```
Comment (MadEste) [inline Lesson02/calculator.js:11]: Nice work, You may also want to tyr js template literals for string concatenation.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: Lesson02/countdown.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+var x = 10;
+
+for (x; x >= 0; x--) {
```
Comment (MadEste) [inline Lesson02/countdown.js:3]: can also declare x within the for loop. While not incorrect let is generally preferred to car nowadays.

https://medium.com/podiihq/javascript-variables-should-you-use-let-var-or-const-394f7645c88f

File: Lesson03/exercise-one.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+let list = ['apple', 'banana', 'carrot', 'daikon', 'eggplant'];
+console.log('Initial array created');
+console.log(list);
+list.push('fruit');
+console.log('New element ' + list[list.length -1] + " added to end of array.");
```
Comment (MadEste) [inline Lesson03/exercise-one.js:5]: 👍

File: Lesson03/exercise-three.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+let pets = ['cats', 'dogs', 'birds', 'rabbits', 'pigs'];
+pets.forEach(e => (console.log(pets.indexOf(e) + ", " + e)));
```
Comment (MadEste) [inline Lesson03/exercise-three.js:2]: Works great, extra respect for using .forEach with an arrow function!

File: Lesson05/script.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
+// function bank for lesson 5 assignment
+function bank() {
+    let balance = 2000;
+    let userCommand = "";
+    while (userCommand != 'q') {
+        userCommand = prompt("Welcome to Bells Pargo Bank \n D - deposit \n W - withdrawal \n B - balance \n Q to quit").toLowerCase();
```
Comment (MadEste) [inline Lesson05/script.js:6]: 👍 Good use of allowing both cases.

File: Lesson05/script.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+            try {
+                switch(userCommand) {
+                    case 'q':
+                        alert('Thanks for Visiting');
+                        break;
+                    case 'w':
+                        let withdrawalAmount = parseFloat(prompt('Withdrawal Amount: ', 0));
+                        if (isNaN(withdrawalAmount)) {
```
Comment (MadEste) [inline Lesson05/script.js:14]: Good work checking if input is number and valid.

**Pull request closed without merging**